.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: required
+
+services:
+  - docker
+
+language: python
+
+python:
+  - "3.7"
+
+install:
+  - pip install pipenv
+  - pipenv install --dev --deploy
+
+script:
+  - make build
```

Comment:
This should be 
```shell-script
make lint
make docker
```
Build only does a pipenv install which is already done in the install step